src/frontend/src/lib/schema/post-message.schema.ts

```diff
@@ -97,11 +108,14 @@ export const PostMessageResponseSchema = z.enum([
 	'syncIcpWallet',
 	'syncIcrcWallet',
 	'syncBtcWallet',
+	'syncSolWallet',
 	'syncIcpWalletError',
 	'syncIcrcWalletError',
 	'syncBtcWalletError',
+	'syncSolWalletError',
 	'syncIcpWalletCleanUp',
 	'syncIcrcWalletCleanUp',
+	'syncSolWalletError',
```

Comment:
`syncSolWalletError` is already there, I think this one was supposed to be `syncSolWalletCleanUp`

Reply:
ah, no i don't think cleanup is used atm. removed duplicate